migration/src/m20230622_104142_stripe_deposits.rs

```diff
@@ -0,0 +1,104 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        // Replace the sample below with your own migration scripts
+        manager
+            .create_table(
+                Table::create()
+                    .table(StripeIncreaseBalanceReceipt::Table)
+                    .if_not_exists()
```

Comment:
Remove the "not_exists" checks. We do our locking such that all steps should always succeed

Reply:
fixed